hudi-common/src/test/java/org/apache/hudi/common/testutils/minicluster/ZookeeperTestService.java

```diff
@@ -163,6 +163,8 @@ private static void setupTestEnv() {
     // resulting in test failure (client timeout on first session).
     // set env and directly in order to handle static init/gc issues
     System.setProperty("zookeeper.preAllocSize", "100");
+    System.setProperty("zookeeper.maxCnxns", "60");
+    System.setProperty("zookeeper.4lw.commands.whitelist", "*");
```

Comment:
same question as above. if it's a test issue then maybe add it in a separate PR?